packages/core/src/amazonqFeatureDev/util/i18n-helper.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import translations from '../../../package.nls.json'
```

Comment:
this file feels like it can live in a more general place. Maybe core/src/shared?

Reply:
I've reused the default file for Q  extension, but perhaps we can change it to something more sharable. I'll double check on that.